ports/crysalis/port.json

```diff
@@ -0,0 +1,31 @@
+{
+  "version": 3,
+  "name": "crysalis.zip",
+  "items": [
+    "Crysalis.sh",
+    "crysalis"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Crysalis",
+    "porter": [
+      "TheLaggyPyro",
+      "kotzebuedog"
+    ],
+    "desc": "Crysalis – the water of life. For generations, people lived their lives, blessed by the divine water. After finding out from the king that the town Ciuna’s Crysalis spring has run dry, Mio, a Crysalis researcher, is called in.\n\nGo on an epic journey with Mio, and friends. Beautiful maps, elegant music, and fun areas to explore!",
+    "desc_md": null,
+    "inst": "Copy Crysalis.exe (extract it from crysalis.zip) and RPGVXAce_RTP.zip in the roms/ports/crysalis folder.",
+    "inst_md": null,
+    "genres": [
+      "rpg"
+    ],
+    "image": null,
+    "rtr": false,
```

Comment:
Is this game r2p?

Reply:
It's not rtr but you make me think that I've put license files (rpgmaker RTP and music) for stuffs that the user has to import by himself. I'll check with Laggy ;)

Reply:
So, Laggy and I came to the conclusion that it won't be rtr (kind of hard to get in touch with the author of this 2011 game). I've removed the useless license files in commit 
1ca9e36